src/JsFunctionsScanner.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace WP_CLI\I18n;
+
+use Gettext\Translations;
+use Gettext\Utils\JsFunctionsScanner as GettextJsFunctionsScanner;
+use Gettext\Utils\ParsedComment;
+use Peast\Peast;
+use Peast\Syntax\Node\Node;
+use Peast\Syntax\Node\VariableDeclaration;
+use Peast\Traverser;
+use Peast\Syntax\Node\CallExpression;
+use Peast\Syntax\Node\Identifier;
+use Peast\Syntax\Node\Literal;
+
+class JsFunctionsScanner extends GettextJsFunctionsScanner {
```

Comment:
Class should be `final` and protected methods/properties should be private (unless they were already protected in the superclass).